src/sass/components/_post-box.scss

```diff
@@ -51,4 +59,21 @@
       padding-top: 30px;
     }
   }
+
+}
+
+.post-box {
+  &:hover {
```

Comment:
You have another `.post-box` defined at line 8. Please move these rules inside the one above.